### DISCUSSION TITLE
View moderator actions in chat (Mod Logs on channel you are not one)

### DISCUSSION THREAD

**Discussion by Pafon22** [night/betterttv#6244] [Ideas]
Like the Emote Menu icon, there could be a Mod Logs with the moderation actions, who gave the timeout, ban, deleted message and who received the punishment (displaying the original message that was censored) in channels that I am not a moderator.
It may not be possible to see which moderator was responsible for the actions, but the rest is still good.

An example I saw: 
https://prnt.sc/QF4r5fN_i18b

In the side menu, you choose whether to see the moderation actions or the log of messages sent by a moderator.

**Comment by night** [comment]
Closing as a duplicate of #6037. Please add your upvote and any extra detail over there so feedback stays consolidated.
